src/FullScreenDialog/FullScreenDialog.types.ts

```diff
@@ -0,0 +1,6 @@
+export type FullScreenDialogProps = {
+  condition: boolean;
+  onClose: () => void;
+  children: React.ReactNode;
+  dialogTitle?: string;
```

Comment:
It would be better to just name this `title` rather than `dialogTitle`. In general we try to stick to single word prop names where possible. There need to be deviations in some cases but in this case title is fine because the componment has dialog in it's name. If it was a button component we wouldn't name the prop buttonColor we would just name it color etc.

Also I think technically this title can actually be any React.ReactNode and it would be rendered.